pkg/onchain/website/uploader.go

```diff
@@ -143,7 +143,7 @@ func uploadMissedChunks(client *node.Client, addr []byte, chunks []string, misse
 		chunkID, err := strconv.Atoi(arrMissedChunks[index])
 		if err != nil {
 			return nil,
-				fmt.Errorf("error converting string to integeter '%w'", err)
+				fmt.Errorf("error converting string to integer '%w'", err)
 		}
 
 		param, err := json.Marshal(AppendParams{
```

Comment:
use appendParams function here, store it into a variable so you can use it again in fmt.Errorf bellow

Reply:
ok !